polar-core/src/polar.rs

```diff
@@ -180,6 +180,18 @@ impl Polar {
         self.load(src, None)
     }
 
+    /// Clear rules from the knowledge base
+    pub fn clear_rules(&self) -> PolarResult<()> {
+        let mut kb = self.kb.write().unwrap();
+        kb.rules = HashMap::new();
+        kb.sources = Sources::default();
+        kb.inline_queries = vec![];
```

Comment:
Why do these three reassign and the following two call `clear()` on the existing data structure? Why not call `clear()` on all five (or reassign all five)?

Reply:
Updated to be consistent. `Sources` still needs to be reassigned to the default.